### PR TITLE
fix(docs): documentation for noUnnecessaryContinue

### PR DESCRIPTION
**Pull request by aravind1078**
## Summary

documentation in website for `noUnnecessayContinue` seems to broken because of indentation, fixed it!

### CODE REVIEW

File: website/src/docs/lint/rules/js/noUnnecessaryContinue.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
+---
+title: Lint Rule js/noUnnecessaryContinue
+layout: layouts/rule.liquid
+showHero: false
+description: disallow unnecessary continue statement inside the loop
```
Comment (ematipico) [inline website/src/docs/lint/rules/js/noUnnecessaryContinue.md:5]:
```suggestion
description: disallow unnecessary `continue` statements inside loops
```

File: website/src/docs/lint/rules/js/noUnnecessaryContinue.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-				key: lint-rules/js/noUnnecessaryContinue
-				parent: lint-rules
-				title: js/noUnnecessaryContinue
-			---
-
-			# js/noUnnecessaryContinue
+# js/noUnnecessaryContinue
 
-			continue statement is unnecessary as the last statement inside the loop, can be safely removed
+`continue` statement is unnecessary as the last statement inside the loop, can be safely removed
```
Comment (ematipico) [inline website/src/docs/lint/rules/js/noUnnecessaryContinue.md:14]:
```suggestion
This rule detects `continue` statements that can be marked as unnecessary. These statements can be safely removed.
```

**Review by ematipico — CHANGES_REQUESTED** [review]
Some wording review

File: website/src/docs/lint/rules/index.md [hunk 3]
```diff
@@ -230,7 +230,7 @@ MISSING DOCUMENTATION
 	<a href="/docs/lint/rules/js/noUnnecessaryContinue">noUnnecessaryContinue</a>
 	<a class="header-anchor" href="#noUnnecessaryContinue"></a>
 </h3>
-unnecessary continue statement inside the loop
+disallow unnecessary continue statement inside the loop
```
Comment (aravind1078) [inline website/src/docs/lint/rules/index.md:233]:
```suggestion
disallow unnecessary continue `statement` inside the loop
```

File: website/src/docs/lint/rules/index.md [hunk 4]
```diff
@@ -230,7 +230,7 @@ MISSING DOCUMENTATION
 	<a href="/docs/lint/rules/js/noUnnecessaryContinue">noUnnecessaryContinue</a>
 	<a class="header-anchor" href="#noUnnecessaryContinue"></a>
 </h3>
-unnecessary continue statement inside the loop
+disallow unnecessary continue `statement` inside the loop
```
Comment (aravind1078) [inline website/src/docs/lint/rules/index.md:233]:
```suggestion
disallow unnecessary `continue` statement inside the loop
```

**Review by ematipico — APPROVED** [review]

**Pull request merged**